From de1b3a5ac3991f5cfb9278c2887c0be2b71dc58b Mon Sep 17 00:00:00 2001
From: Luiz Augusto von Dentz <luiz.von.dentz@intel.com>
Date: Fri, 8 Nov 2024 13:15:10 -0500
Subject: [PATCH] avrcp: Fix not marking AVRCP_EVENT_VOLUME_CHANGED as
 supported

If avrcp_volume_supported return true then AVRCP_EVENT_VOLUME_CHANGED
shall be marked as supported.

Fixes: https://github.com/bluez/bluez/issues/1007
---
 profiles/audio/avrcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/profiles/audio/avrcp.c b/profiles/audio/avrcp.c
index b5e6548e66..9fe8f55e5d 100644
--- a/profiles/audio/avrcp.c
+++ b/profiles/audio/avrcp.c
@@ -4268,7 +4268,7 @@ static void target_init(struct avrcp *session)
 	if (target->version < 0x0104)
 		return;
 
-	if (!avrcp_volume_supported(target))
+	if (avrcp_volume_supported(target))
 		session->supported_events |=
 				(1 << AVRCP_EVENT_VOLUME_CHANGED);
 
